scripts/check-peer-deps.js

```diff
@@ -61,12 +62,21 @@ async function main() {
   const extraneousInPeerDeps = peerDependencies.filter((name) => {
     return !expectedPeerDependencies.has(name);
   });
+  const peerDependenciesWithoutReact = peerDependencies.filter((name) => {
+    return !excludedFromProdDependencies.has(name);
+  });
+  const extraneousInProdDeps = peerDependencies.find((name) => {
```

Comment:
Should this be filter, not find? Otherwise it seems to fail when I'm running this locally

```suggestion
  const extraneousInProdDeps = peerDependencies.filter((name) => {
```